### PR TITLE
Add heat-operator kuttl tests in e2e job

### PR DESCRIPTION
**Pull request by jparoly**
Add step to run kuttl tests.
Add new image that installs kuttl and skopeo which are needed to run the kuttl tests

### CODE REVIEW

**Comment by jparoly** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-heat-operator-main-heat-operator-build-deploy-kuttl

File: ci-operator/jobs/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main-presubmits.yaml
Comment (cescgina) [inline ci-operator/jobs/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main-presubmits.yaml:1]: In this file, we should set `always_run: false` for the `operator-build` job (line 4), as the build-deploy-kuttl job already has the build step

Comment (jparoly) [reply ci-operator/jobs/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main-presubmits.yaml:1]: thank you, done

**Comment by cescgina** [comment]
Changes looks good to me, but it seems that you need to run `make jobs` again

**Comment by cescgina** [comment]
/pj-rehearse pull-ci-openstack-k8s-operators-heat-operator-main-heat-operator-build-deploy-kuttl

File: ci-operator/config/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main.yaml [hunk 1]
```diff
@@ -63,6 +75,17 @@ tests:
   container:
     from: openstack-operator-ci-pre-commit
   optional: true
+- as: heat-operator-build-deploy-kuttl
```
Comment (Sandeepyadav93) [inline ci-operator/config/openstack-k8s-operators/heat-operator/openstack-k8s-operators-heat-operator-main.yaml:78]: We can get rid of `heat-operator-build` job in same PR, As kuttl job will already build the operator PR first.

Earlier we don't have kuttl tests and heat was not intergrated in meta-operator so we were just running operator build job.

**Comment by jparoly** [comment]
Handled by https://github.com/openshift/release/pull/40877 now

**Pull request closed without merging**